.gitignore

```diff
@@ -5,12 +5,12 @@
 save*.yaml
 *~
 *#
-/test-resources/RobotBuilderTestProject/*
+/test-resources/
 /.svn/*
 *.exec
 dependency-reduced-pom.xml
 nbactions.xml
-
+src/main/resources/export/vendordeps/
```

Comment:
These files should probably be put into `"${buildDir}/generated/resources/"` rather than gitingored in the src directory. This is how we handle the WPILibVersion.java in allwpilib.

Reply:
move the location, left the gitignore entry for existing files